sklearn/metrics/scorer.py

```diff
@@ -313,12 +322,12 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 # Standard regression scores
 r2_scorer = make_scorer(r2_score)
-mean_squared_error_scorer = make_scorer(mean_squared_error,
-                                        greater_is_better=False)
-mean_absolute_error_scorer = make_scorer(mean_absolute_error,
-                                         greater_is_better=False)
-median_absolute_error_scorer = make_scorer(median_absolute_error,
-                                           greater_is_better=False)
```

Comment:
Because those variables do not have a leading `_` the can unfortunately be considered public API even if not in the documentation. I think we should maintain backward compat aliases until 0.20:

```python
# Backward compat alias to keep until the end of the deprecation period (0.20)
mean_squared_error_scorer = neg_mean_squared_error_scorer
mean_absolute_error_scorer = neg_mean_absolute_error_scorer
median_absolute_error_scorer = neg_median_absolute_error_scorer
```